### PR TITLE
Added response fields descriptions table

### PR DESCRIPTION
**Pull request by aviveldan**
@BenAlfasi @mjcaparas

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md [hunk 1]
```diff
@@ -93,3 +93,12 @@ Content-type: application/json
...
 }
 ```
+
+
+Name | Description
+:---|:---|:---
+Org prevalence  | the distinct count of devices that opened network connection to this IP.
+Org first seen  | the first connection for this IP in the organization.
+Org last seen  | the last connection for this IP in the organization.
+>[!Note]
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md:103]: Minimum 1 blank line spacing needed between a table and any subsequent non-table element:
```suggestion

> [!Note]
```
(also added recommended MarkDown indent marker compatibility spacing)

File: windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md [hunk 2]
```diff
@@ -93,3 +93,12 @@ Content-type: application/json
     "orgLastSeen": "2017-08-29T13:32:59Z"
 }
 ```
+
+
+Name | Description
+:---|:---|:---
+Org prevalence  | the distinct count of devices that opened network connection to this IP.
+Org first seen  | the first connection for this IP in the organization.
+Org last seen  | the last connection for this IP in the organization.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md:102]: Also, when creating MarkDown tables, it helps to follow a basic standard to make sure the table is correct.
```suggestion

| Name | Description |
| :--- | :---------- |
| Org prevalence | the distinct count of devices that opened network connection to this IP. |
| Org first seen | the first connection for this IP in the organization. |
| Org last seen  | the last connection for this IP in the organization. |
```
By the way, did you actually intend to add an empty third column?

**Comment by illfated** [comment]
Sorry if my comments seem like critique, they are not meant to sound like that. This being your first MS Docs PR and all, you deserve a thank you for contributing.

Myself, I am trying to be helpful, but I sometimes forget how my phrasing may look to newcomers or others in general.

File: windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md [hunk 3]
```diff
@@ -93,3 +93,14 @@ Content-type: application/json
...
+
+
+
+| Name | Description |
+| :--- | :---------- |
+| Org prevalence | the distinct count of devices that opened network connection to this IP. |
+| Org first seen | the first connection for this IP in the organization. |
+| Org last seen  | the last connection for this IP in the organization. |
+
+> [!Note]
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md:105]: MS codestyle/coherence:
```suggestion
> [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md [hunk 4]
```diff
@@ -93,3 +93,14 @@ Content-type: application/json
     "orgLastSeen": "2017-08-29T13:32:59Z"
 }
 ```
+
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md:97]: 3 blank lines is a bit too much, even for spacing between a code block and a table. 2 should be enough.
```suggestion
```

File: windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md [hunk 5]
```diff
@@ -93,3 +93,14 @@ Content-type: application/json
...
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/get-ip-statistics.md:98]: 3 consecutive blank lines is a tad much, even for spacing between a code block and a table. 2 should be enough.
```suggestion
```

**Comment by illfated** [comment]
Aside from the cosmetics and codestyle, this looks pretty good to me.

**Comment by e0i** [comment]
cc @JohanFreelancer9 for copy

**Review by JohanFreelancer9 — APPROVED** [review]
Approved with @illfated's suggestions.

**Pull request merged**